Gemfile

```diff
@@ -25,6 +25,9 @@ gem 'cancancan'
 gem 'rgeo'
 gem 'rgeo-geojson'
 gem 'rails_12factor', group: :production
+gem 'high_voltage', github: 'thoughtbot/high_voltage'
+gem 'georuby', require: ['georuby', 'geo_ruby/ewk']
```

Comment:
Right now, I'm leaning towards wanting to remove rgeo as it's provides a lot of extra functionality that we don't use like trying to reproject coordinates.  Where as georuby is much lighter weight and just handles getting reading and writing geospatial data.

I think that would be better done as a separate issue to, as I'll need to update the db:seed stuff